pkg/controller/export/export_config_controller.go

```diff
@@ -111,10 +111,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 	}
 
 	if len(cfg.Data) == 0 {
-		return reconcile.Result{}, fmt.Errorf(fmt.Sprintf("data missing in configmap %s, unable to configure exporter", request.NamespacedName))
+		return reconcile.Result{}, fmt.Errorf("data missing in configmap, unable to configure exporter")
```

Comment:
why do we need to remove the variable?

```suggestion
		return reconcile.Result{}, fmt.Errorf("data missing in configmap %s, unable to configure exporter", request.NamespacedName)
```

Reply:
To prevent hitting this error - `non-constant format string in call to fmt.Errorf` - https://github.com/open-policy-agent/frameworks/actions/runs/14718290117/job/41306682777?pr=552#step:8:640.

Reply:
The issue is that the existing code is using fmt.Sprintf to create a formatted string with a variable, which results in a dynamically constructed format string that is passed to fmt.Errorf, which is disallowed for the format string of fmt.Errorf.  Remove `fmt.Sprintf`  is the right solution, not removing the actual variable from the error, which we need.
e.g. https://go.dev/play/p/NG_sgyuMtDB

Reply:
Ahh, ok. Yeah I tested this with 1.23 go version in the mod again and the error didnt appear so I thought that "variable" string is an issue and in 1.24 only static string is expected. But this makes sense. Updated to suggested change here.